guides/common/attributes-satellite.adoc

```diff
@@ -93,6 +92,7 @@
 :PIV: CAC
 :project-context: satellite
 :project-change-hostname: satellite-change-hostname
+:project-installer-package: satellite
```

Comment:
```suggestion
:project-installer-package: {project-context}
```

Reply:
I disagree here. It's very specific that package name and it's not like something else inherits from the satellite attributes.

I'll also note we (installer team) are rethinking the packaging so that the end result is that you only have a single scenario present (think `foreman-installer-$scenario` packages). That way you never have to specify `--scenario` to the installer (just like in vanilla Foreman). These attributes will make the transition easier in the future